filechooser-portal/Main.vala

```diff
@@ -45,6 +47,12 @@ public class Files.FileChooserPortal : Object {
         out uint response,
         out HashTable<string, Variant> results
     ) throws DBusError, IOError {
+        if (parent_window in dialogs) {
+            results = new HashTable<string, Variant> (null, null);
+            response = 2;
```

Comment:
Is there some constant we can use for this to be clear about what `2` means in this context?

Reply:
I can't find anything, both the xdg-desktop-portal and xdg-desktop-portal-gtk use the numbers directly, `2` seems to be when the  dialog is closed without an response.